src/app/app.component.ts

```diff
@@ -8,4 +8,10 @@ import {Router} from '@angular/router';
 })
 export class AppComponent {
   title = 'ToDoListApp';
+
+  constructor(private userDataService: UserDataService) {}
+
+  public isLogining():boolean{
+    return this.userDataService.isLogining();
```

Comment:
Please use here the same method as you are using in AuthGuard to check whether a user is authenticated

Reply:
used